src/screens/Details/components/Header/header.tsx

```diff
@@ -0,0 +1,31 @@
+import {View, StyleSheet} from 'react-native';
+import BackArrow from '../../../../assets/icons/Arrow.svg';
```

Comment:
can be useful

https://medium.com/@enzzorp/setting-up-aliases-in-react-native-de6c73b0dc5f